psa/Makefile

```diff
@@ -0,0 +1,33 @@
+CC       = gcc
+LIB_PATH = /usr/local
+CFLAGS   ?= -Wall -I$(INCLUDE_PATH)
+LIBS     ?= -L$(LIB_PATH)/lib -lm
+
+# PSA implementation
+PSA_INCLUDE ?=
+PSA_LIB_PATH ?=
+
+# option variables
+DYN_LIB         = -lwolfssl
```

Comment:
Please add `-lm`. If DH is not using `WOLFSSL_DH_CONST` then it will use math `pow` and `log`.

```
$ make
gcc -o client-tls13-ecc-psa client-tls13-ecc-psa.c -Wall -I -Os -I/tmp/mbedtls/build/include -L/usr/local/lib -lm -lwolfssl /tmp/mbedtls/build/library/libmbedcrypto.a
/usr/bin/ld: /usr/local/lib/libwolfssl.so: undefined reference to `log'
/usr/bin/ld: /usr/local/lib/libwolfssl.so: undefined reference to `pow'
collect2: error: ld returned 1 exit status
make: *** [Makefile:31: client-tls13-ecc-psa] Error 1
```